internal/tools/orchestrator/hepa/providers/domain/domain.service.go

```diff
@@ -117,9 +123,67 @@ func (s *domainService) ChangeInnerIngress(ctx context.Context, req *pb.ChangeIn
 	}
 	return
 }
+
+func (s *domainService) getUintOrgId(ctx context.Context) (uint64, error) {
+	oId, err := apis.GetIntOrgID(ctx)
+	if err != nil {
+		return 0, err
+	}
+	return uint64(oId), nil
+}
+
+func (s *domainService) preCheckPermission(userID string, services *bundle.GetRuntimeServicesResponseData) error {
+	var (
+		appId        = services.Extra.ApplicationId
+		resourceType = s.getPermissionResource(services)
+	)
+
+	if userID == "" {
+		return fmt.Errorf("not login")
```

Comment:
Confirm whether the built-in API error 'NotLogin' exists.

Reply:
it doesn't exists